virt-v2v/cold/entrypoint.go

```diff
@@ -127,6 +128,25 @@ func buildCommand() []string {
 				virtV2vArgs = append(virtV2vArgs, "--mac", macToIp)
 			}
 		}
+		// Adds LUKS keys, if exist.
+		if _, err := os.Stat(LUKSDIR); os.IsNotExist(err) {
+			// do nothing
```

Comment:
that depends on when we wish to have a real error/message.
if the directory isn't there, it might mean the user don't user LUKS. so why should we error/log?

Reply:
> if the directory isn't there, it might mean the user don't user LUKS. so why should we error/log?

I don't have an answer for this - I didn't think of raising an error or writing a log here

what I wrote above is to handle only the case of `!os.IsNotExist(err)` (in other words - not doing anthing when `os.IsNotExist(err)`)

Reply:
i think the better way is as it is now (in terms of error/log). but in case you have strong opinion to change it, i will.